src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/OAuth2/TokenResponse.cs

```diff
@@ -108,6 +109,9 @@ internal class TokenResponse
         [DataMember(Name = TokenResponseClaim.Claims, IsRequired = false)]
         public string Claims { get; set; }
 
+        [DataMember(Name = TokenResponseClaim.ClientInfo, IsRequired = false)]
+        public string ClientInfo { get; set; }
```

Comment:
any backward and forward compatibility testing needed?

Reply:
no, since this DataMember is optional

Reply:
Imo: a good practice to ensure this is utested at least.

---
In reply to: [158566106](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/925#discussion_r158566106) [](ancestors = 158566106)

Reply:
it is tested indirectly since many tests  use MockHttpMessageHandler were token response is provided in the form of string so it is deserialized as TokenResponse object